src/pages/admin/pages/faq/lib/popularCategories.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": 1,
+    "icon": "diversity_1",
```

Comment:
That's not the icon in design

Design:
<img width="199" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/dc024065-74bc-4e2e-af5a-3fc390bf954c">

PR:
<img width="177" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/dac820e5-5b0f-4f2f-b5d7-de3515540add">

I see it being available in font. Maybe we use wrong version?
<img width="480" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/39f420a6-2c14-4482-8762-9393d0866d27">